LA3-V2/style.css

```diff
@@ -313,185 +310,130 @@ hr {
   margin-top: 0.5em;
 }
 
-@media (max-width: 1366px) {
-  nav {
-    padding-left: 0;
-    /* background-color: #09529b; */
-  }
-  
-  h3 {
-    margin-left: 5em;
-  }
-  
-  nav ul li {
-    margin-left: 3em;
-  }
-  
-  nav ul li:first-child {
-    margin-left: 5em;
-  }
-  
-  .btn-sign {
-    margin-right: 7em;
-  }
-  
-
-  /* MAIN CONTENT  */
-  .content {
-    margin-left: 4em;
-  }
-
-  .title {
-    width: fit-content;
-  }
-
-  .text{
-    width: 100%;
-    text-align: justify;
-  }
-
-  /* FOOTER  */
-  .footer-content {
-    padding-left: 1em;
-    padding-right: 2em;
-    padding-bottom: 1em;
-  }
-
-  .footer-title-company{
-    font-size: 20px;
-    padding-top: 1em;
-  }
-
-  .company {
-    margin-left: 3em;
-    margin-right: 3em;
-  }
-  
-  .footer-title {
-    font-size: 20px;
-  }
-  
-  .text-list li{
-    padding-bottom: 7px;
-  }
-
-  .footer-newsletter{
-    margin-top: 1.5em;
-    margin-left: 14em;
-  }
-
-  .form{
-    text-indent: 7px;
-  }
-}
-
-@media (max-width: 936px) {
-  h3 {
-    margin-left: 3em;
```

Comment:
Why delete? There's no need to delete the media query for the different max width resolutions